code/modules/antagonists/heretic/knowledge/side_void_blade.dm

```diff
@@ -107,22 +107,6 @@
 	demolition_mod = 1.5
 	sharpness = SHARP_EDGED
 
-/obj/item/mutant_hand/shattered_risen/Initialize(mapload)
```

Comment:
Is this removal from here to line 125 intentional?

Reply:
Yes, all of it is copy pasted on the base type